tests/appsec/test_asm_standalone.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+from utils import weblog, interfaces, scenarios, features
+from utils._context.header_tag_vars import *
+
+
+class Test_AppSecPropagation:
+    """APM correctly propagates AppSec events in distributing tracing."""
+
+    def setup_trace_no_upstream_appsec_propagation_no_attack_is_dropped(self):
+        trace_id = 1212121212121212121
+        parent_id = 34343434
+        self.r = weblog.get(
+            "/waf/",
+            headers={
+                "x-datadog-trace-id": str(trace_id),
+                "x-datadog-parent-id": str(parent_id),
+                "x-datadog-origin": "rum",
+                "x-datadog-tags": "_dd.p.other=1",
+            },
+        )
+
+    @scenarios.appsec_standalone
```

Comment:
If all your tests belong to this scenario, you can set the decorator directly on the class